pkg/plugins/runtime/k8s/webhooks/owner_reference_mutator.go

```diff
@@ -77,3 +84,11 @@ func (m *OwnerReferenceMutator) Handle(ctx context.Context, req admission.Reques
 	}
 	return admission.PatchResponseFromRaw(req.Object.Raw, mutatedRaw)
 }
+
+func syncedResource(cpMode config_core.CpMode, labels map[string]string) bool {
+	syncedOrigin := mesh_proto.GlobalResourceOrigin
+	if cpMode == config_core.Global {
+		syncedOrigin = mesh_proto.ZoneResourceOrigin
+	}
+	return len(labels) > 0 && labels[mesh_proto.ResourceOriginLabel] == string(syncedOrigin)
```

Comment:
you can use [ResourceOrigin](https://github.com/kumahq/kuma/blob/b0789c8fda5bd8d8724cd06b5c97de049e22c6c4/pkg/core/resources/model/resource.go#L443) func

Reply:
it takes meta. I don't have a core resource here, only kube resource